Makefile

```diff
@@ -68,9 +67,6 @@ test-resolc: install
 test-workspace: install
 	cargo test --workspace --exclude revive-llvm-builder
 
-test-cli: install
-	npm run test:cli
```

Comment:
We could also keep the `make test-cli` and have it run the Rust tests (perhaps with `cargo test tests::cli`), in order to keep the short and sweet way of running only those.